content/company/strategy/code-graph/code-intelligence/index.md

```diff
@@ -45,15 +45,16 @@ At the moment maturity varies depending on the area of ownership:
 
 - **Language indexers:** Maturity varies widely per language. See [LSIF indexers documentation](https://docs.sourcegraph.com/code_intelligence/references/indexers).
 
-- **Code intelligence platform:** Viable with a solid foundation, we're currently working on scaling and improving performance to push the limits of precise indexing and build the Global Code Graph.
+- **Code intelligence platform:** We're currently focusing on delivering an out-of-the-box precise code intelligence solution to a small number of eager customers.
```

Comment:
Can we define a bit more what an out of the box precise code intelligence solution is? Does that mean easy to setup? Or lots of languages? Something else?

Reply:
```suggestion
- **Code intelligence platform:** We're currently focusing on delivering an out-of-the-box precise code intelligence solution to a small number of eager customers. This will allow customers to easily set up precise code intelligence for Go, Java, Scala, Kotlin, and soon TS repositories. This feature will also enable automatic indexing of a repository's dependencies, which means users will now be able to navigate and find references across repositories and dependencies alike. 
```